ArduPlane/servos.cpp

```diff
@@ -902,6 +903,26 @@ void Plane::servos_output(void)
     quadplane.tailsitter_output();
     quadplane.tiltrotor_bicopter();
 
+     // pull up tilts and set throttle_min if LANDING_FLARE RCx_OPTION switch is set and in FW mode, manual throttle
+     if (!quadplane.in_transition() && !control_mode->is_vtol_mode() && !auto_throttle_mode && plane.channel_throttle->norm_input() < -0.95 && plane.flare_switch_active) {
+         int32_t tilt = -4500;
```

Comment:
You can use the #define 
```suggestion
         int32_t tilt = -SERVO_MAX;
```

Reply:
done